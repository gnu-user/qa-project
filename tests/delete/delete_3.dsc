TITLE
    Delete User Input (Invalid User)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the delete 
    transaction, and enter a username which does not exist. Verify that the 
    username entered is rejected, and an error occurs.
DEPENDENCIES
    login Test Suite
    delete_1
