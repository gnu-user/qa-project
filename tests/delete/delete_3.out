Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the username to delete:
Invalid username, username does not exist.